packages/web/src/common/store/cache/collections/sagas.js

```diff
@@ -473,7 +506,13 @@ function* confirmAddTrackToPlaylist(userId, playlistId, trackId, count) {
           )
         )
       },
-      result => (result.playlist_id ? result.playlist_id : playlistId)
+      result => (result.playlist_id ? result.playlist_id : playlistId),
+      undefined,
+      {
+        operationId: 'ADD_TRACK',
```

Comment:
I wonder if we could infer ADD_TRACK from something? or at least pull it in from an action type or something to make it bound

Reply:
ohh yeah ill explore this a it. Although, we might want to keep this free so that multiple actions can use the same operation id (if whatever confirm calls they make can be grouped together)

Reply:
usually i'm no enum, but enum might be nice for op id